bdk-ffi/Cargo.toml

```diff
@@ -18,10 +18,10 @@ path = "uniffi-bindgen.rs"
 default = ["uniffi/cli"]
 
 [dependencies]
-bdk_wallet = { version = "=1.0.0-beta.6", features = ["all-keys", "keys-bip39", "rusqlite"] }
-bdk_core = { version = "0.4.0" }
-bdk_esplora = { version = "0.20.0", default-features = false, features = ["std", "blocking", "blocking-https-rustls"] }
-bdk_electrum = { version = "0.20.0", default-features = false, features = ["use-rustls-ring"] }
+bdk_wallet = { version = "=1.0.0", features = ["all-keys", "keys-bip39", "rusqlite"] }
```

Comment:
I think we should be able to remove the `=` sign here and let cargo do its thing since new feature versions should not introduce breaking changes.

Reply:
updated to remove the `=`